<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="TableWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>681</width>
      <height>261</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="getDataFromFile">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>290</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>ReadDate</string>
    </property>
   </widget>
   <widget class="QPushButton" name="saveBtn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>330</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Сохранить</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addNext">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>290</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Следующий день</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addPrev">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>290</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Предыдущий день</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addIsLeap">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>330</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Високосный год</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addWeak">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>330</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Недели</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addDur">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>330</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Длительность</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addBith">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>430</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Добавить др</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addDate">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>390</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Добавить дату</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addCurrDate">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>290</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Текущая дата</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditAddItem">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>390</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditBirth">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>430</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
